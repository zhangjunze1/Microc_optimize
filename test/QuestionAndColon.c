void main() {
  int n=5;
  if(n? 1:0){
    print("%d",n);
  }
}
