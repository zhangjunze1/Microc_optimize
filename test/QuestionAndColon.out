24 19 0 5 25 13 0 5 12 13 11 17 17 0 1 16 19 0 0 17 26 13 11 22 21 1 21 0